ICON_EMBEDDING.md

```diff
@@ -51,7 +51,7 @@ If you prefer to run the commands manually:
 
 ```bash
 # Convert SVG to ICO (requires ImageMagick)
-magick convert -background none assets/logo-icon.svg -define icon:auto-resize=256,128,64,48,32,16 assets/logo-icon.ico
```

Comment:
Please revert this if you can, it has to be `magick convert` for future ImageMagick versions to work (since v7 just calling `convert` standalone is deprecated!)